### DISCUSSION TITLE
Backup and Restore Not working in Openshift Cluster with Minio and Velero

### DISCUSSION THREAD

**Discussion by MuthuSundaravadivel** [vmware-tanzu/velero#3254] [Community support Q&A]
I am trying to take the backup of PVC in openshift cluster using minio and velero but the backup and restore not happening properly. Can you please help me resolve the issue

Velero Installation:

velero install
--provider aws
--plugins velero/velero-plugin-for-aws:v1.0.0
--bucket velero
--secret-file ./credentials-velero
--use-volume-snapshots=true
--use-restic —wait
--default-volumes-to-restic
--backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000

POD Annotation:
oc -n dev-ms annotate pod/mongodb-1-dqkd6 backup.velero.io/backup-volumes=mongodb-data,default-token-npk2z

Backup:
velero backup create mongodb --include-cluster-resources=true --include-namespaces=dev-ms
Restore:
velero restore create mongodb --from-backup mongodbackup

<img width="893" alt="Screenshot 2020-11-17 at 4 44 28 PM" src="https://user-images.githubusercontent.com/5441770/99482897-0bc75700-2983-11eb-8e92-d2bbd1bf826d.png">
<img width="895" alt="Screenshot 2020-11-18 at 9 41 21 AM" src="https://user-images.githubusercontent.com/5441770/99482899-0ec24780-2983-11eb-9c87-a1a8b407a2f2.png">

**Comment by dsu-igeek** [comment]
Is Minio installed in the same cluster?  If not, the minio.velero.svc address will not work as it's local to the Kubernetes cluster and you will need to specify the public (at least in your network) name or IP address to contact it.  More info would help us understand the issue if that's not the case.

**The output of the following commands will help us better understand what's going on**:
(Pasting long output into a [GitHub gist](https://gist.github.com) or other pastebin is fine.)

- `kubectl logs deployment/velero -n velero`
- `velero backup describe <backupname>` or `kubectl get backup/<backupname> -n velero -o yaml`
- `velero backup logs <backupname>`
- `velero restore describe <restorename>` or `kubectl get restore/<restorename> -n velero -o yaml`
- `velero restore logs <restorename>`


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Velero version (use `velero version`):
- Velero features (use `velero client config get features`):
- Kubernetes version (use `kubectl version`):
- Kubernetes installer & version:
- Cloud provider or hardware configuration:
- OS (e.g. from `/etc/os-release`):
